pycfmodel/model/generic.py

```diff
@@ -31,6 +32,22 @@ class _Auxiliar(BaseModel):
         ResolvableStrOrList,
     ]
 
+    @validator("*", pre=True)
```

Comment:
```suggestion
    @validator("aux", pre=True)
```

Reply:
Fixed in [1d7b840](https://github.com/Skyscanner/pycfmodel/pull/97/commits/1d7b8400db7709cc8289918937a64a24f2e82125)